scripts/generic-ci-tests.sh

```diff
@@ -151,7 +151,10 @@ case "$TEST_SUITE" in
 
     "js-unit")
         $TOX paver test_js --coverage
-        $TOX paver diff_coverage
+        if [ "$TEST_SERVICE" != "gh-actions" ]
+        then
+            $TOX paver diff_coverage
```

Comment:
I'd like to hear more about the current problem with running this in GHA.

Reply:
This exclusion is removed, diff_coverage is working fine now.